core-metastore/src/main/java/org/zalando/nakadi/repository/db/SubscriptionDbRepository.java

```diff
@@ -121,28 +124,30 @@ public void deleteSubscription(final String id)
     }
 
     /**
-     * Use {@code SubscriptionTokenLister} instead
+     * Internal use only.
+     */
+    public List<Subscription> listAllSubscriptionsFor(final Set<String> eventTypes)
+            throws ServiceTemporarilyUnavailableException {
+
+        return listSubscriptions(eventTypes, Optional.empty(), Optional.empty(), Optional.empty());
+    }
+
+    /**
+     * Use {@code listSubscriptions} with token instead.
      */
     @Deprecated
     public List<Subscription> listSubscriptions(final Set<String> eventTypes, final Optional<String> owningApplication,
```

Comment:
Do I get it correct: this method is `public` + `deprecated` + `listAllSubscriptionsFor` created ? 
Maybe it should not be public or `listSubscriptionsFor` should iterate through token and this method expected to be removed?

Reply:
We cannot remove it until we removed the `offset`-based pagination :(

The new method is only used internally (never via the API calls) and we will keep that one.